src/TextareaAutosize.js

```diff
@@ -3,9 +3,11 @@
  */
 
 import React from 'react';
-import emptyFunction from 'react/lib/emptyFunction';
+import ReactDOM from 'react-dom';
```

Comment:
I guess so, but I am not entirely sure. I theory it should work with older versions but you end up with a duplicate findDOMNode implementation in both react 0.13 and react-dom.